src/pages/home/slider/slider-home-page.js

```diff
@@ -22,6 +24,8 @@ const SliderHomePage = () => {
   const { loading, error } = useQuery(getAllSlides, {
     onCompleted: (data) => setSlides(data.getAllSlides.items)
   });
+  const { sort, page, countPerPage, defaultCount, defaultPage } = URL_QUERIES_NAME;
+  const pathToCatalog = `${pathToCategory}?${sort}=${POPULARITY}&${page}=${defaultPage}&${countPerPage}=${defaultCount}`;
```

Comment:
Actualy I don't understand why you set search params here if you use searchParams? It values set by default in url.
![image](https://user-images.githubusercontent.com/83120263/204153708-d6ba8385-6e14-4faf-9268-d3b4921fe531.png)

Reply:
removed this, it was try to reduce rerenders